sys_conn.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 // OOBCapablePacketConn is a connection that allows the reading of ECN bits from the IP header.
-// If the PacketConn passed to Dial or Listen satisfies this interface, quic-go will use it.
+// If the PacketConn passed to [Dial] or [Listen] satisfies this interface, quic-go will use it.
```

Comment:
Unrelated to this PR, we should have changed this a long time ago.
```suggestion
// If the PacketConn passed to the [Transport] satisfies this interface, quic-go will use it.
```